addons/website_slides/models/slide_slide.py

```diff
@@ -433,6 +455,23 @@ def _compute_youtube_id(self):
             else:
                 slide.youtube_id = False
 
+    @api.depends('video_url', 'video_source_type')
+    def _compute_vimeo_id(self):
```

Comment:
Can't we compute video source type and video ID in the same method ? Just to avoid regexing twice ... Not sure it helps, just asking to check if we can reduce a bit computation complexity.